Description: do not invoke `gn gen` after building gn,
 because this doesn't allow specifying the build arguments in a separate
 args.gn file. `gn gen` is run explicitly afterwards in the override-build
 scriptlet.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -84,10 +84,6 @@ def main(argv):
   gn_gen_args = options.gn_gen_args or ''
   if not options.debug:
     gn_gen_args += ' is_debug=false'
-  subprocess.check_call([
-      gn_path, 'gen', out_dir,
-      '--args=%s' % gn_gen_args, "--root=" + SRC_ROOT
-  ])
 
 
 if __name__ == '__main__':
